deployment/ansible/group_vars/development_template

```diff
@@ -6,6 +6,10 @@ production: false
 # open trip planner vars
 otp_process_mem: 2G
 
+# papertrail logging
+papertrail_host: 'SUBDOMAIN_FOR_PAPERTRAIL_ACCOUNT'
+papertrail_port: 'PORT_FOR_PAPERTRAIL_ACCOUNT'
+
```

Comment:
They're both only set up for production, so I might as well remove papertrail from this template.